v0/destinations/ga4/transform.js

```diff
@@ -59,7 +62,9 @@ function trackResponseBuilder(message, { Config }) {
     case "gtag":
       // gtag.js
       rawPayload.client_id =
```

Comment:
Please add some documentation about client_id

Reply:
Added

Reply:
Not blocking buts a better practice to store this in an independent variable and check for its presence before attaching it to another object